packages/panna-sdk/src/react/components/collectibles/select-recipient-step.tsx

```diff
@@ -38,7 +55,55 @@ export function SelectRecipientStep({ form }: SelectRecipientStepProps) {
           </FormItem>
         )}
       />
-      <Button type="button" onClick={() => next()} disabled={!recipientAddress}>
+      {token.type === TokenERC.ERC1155 && (
+        <FormField
+          control={form.control}
+          name="amount"
+          render={({ field }) => (
+            <FormItem className="flex flex-col gap-2">
+              <div className="flex flex-col gap-3">
+                <div className="flex justify-between">
+                  <FormLabel>Quantity</FormLabel>
+                  <Button
+                    variant="ghost"
+                    size="sm"
+                    type="button"
+                    className="bg-muted hover:bg-border! h-6 p-2"
+                    onClick={() => handleMaxValue()}
+                  >
+                    Max
+                  </Button>
+                </div>
+                <FormControl>
+                  <Input
+                    {...field}
+                    placeholder="Quantity"
+                    type="number"
+                    min={1}
```

Comment:
Reuse the constant here?
```suggestion
                    min={MIN_VALUE}
```